.github/workflows/test.yaml

```diff
@@ -34,7 +34,8 @@ jobs:
         with:
           pipeline_version: v0.41.0
           cli_version: v0.29.1
-
```

Comment:
nit: please revert the changes on this file.

Reply:
Done.